src/components/NavDesktop.js

```diff
@@ -8,6 +8,22 @@ import AccountNav from './AccountNav';
 import AsylumConnectButton from './AsylumConnectButton';
 import FavoritesLink from './FavoritesLink';
 import Language from './Language';
+import '../App.scss'
+
+const transparentSpace = {
```

Comment:
Can you add these styles and the ones from `App.scss` into the styles object below? This keeps styling consistent in the entire app. Here is the documentation for how to use that library https://material-ui.com/styles/basics/